bodhi/server/models.py

```diff
@@ -633,8 +633,14 @@ def get_pkg_committers_from_pagure(self):
         * The second list contains FAS group names that have commit access.
         """
         pagure_url = config.get('pagure_url')
-        # Pagure uses plural names for its namespaces such as "rpms"
-        namespace = self.type.name + 's'
+        # Pagure uses plural names for its namespaces such as "rpms" except for
+        # container
+        # This if statement is not tested because there isn't direct support
+        # for containers in Bodhi. This is just future proofing.
+        if self.type.name == 'container':  # pragma: no cover
```

Comment:
We should use mock to test this code to ensure that it uses the correct namespace. Even though ```self.type``` doesn't know about containers yet, it wouldn't be hard in a test to construct a Package obect and then set it's type attribute to a mock object that have a name attribute that is set to "container".

Also, this commit doesn't seem related to the first commit in the pull request. I recommend splitting this out into a separate pull request to separate concerns. It'll make it easier for you to respond to review comments, and it will also make it easier to review the changes for me. Splitting it is not required, just a recommendation.